ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/route53/private-hosted-zone/provision/cucushift-installer-rehearse-aws-ipi-route53-private-hosted-zone-provision-chain.yaml

```diff
@@ -0,0 +1,13 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-route53-private-hosted-zone-provision
+  steps:
```

Comment:
let's create AWS resources firstly, and then generate the config file:
```
steps:
    - ref: aws-provision-vpc-shared
    - ref: aws-provision-route53-private-hosted-zone
    - ref: ipi-conf
    - ref: ipi-conf-aws-custom-vpc
    - ref: ipi-conf-aws-route53-private-hosted-zone
    - ref: ipi-conf-aws
    - ref: ipi-install-monitoringpvc
    - chain: ipi-install
```

Reply:
done